.github/workflows/build-and-test.yml

```diff
@@ -2,7 +2,7 @@ name: Build and Run Tests
 
 on:
   push:
-    branches: [develop, production]
+    branches: [develop, production, chore/tests/increase-coverage-1]
```

Comment:
Are you trying to do something specific? This should already be covered by the `pull_request` section against `develop`

Reply:
yeah, something specific...removed finally of course